components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -397,21 +389,19 @@ - (void)setEnableRippleBehavior:(BOOL)enableRippleBehavior {
 
 - (CGSize)actionFittingSizeInHorizontalLayout {
   CGSize size = CGSizeZero;
-  UIEdgeInsets insets = self.enableAdjustableInsets ? self.actionsInsets : MDCDialogActionsInsets;
   NSArray<MDCButton *> *buttons = self.actionManager.buttonsInActionOrder;
   if (0 < buttons.count) {
     CGFloat maxButtonHeight = MDCDialogActionButtonMinimumHeight;
-    size.width = insets.left + insets.right;
+    size.width = self.actionsInsets.left + self.actionsInsets.right;
```

Comment:
Is self.actionInsets guaranteed to have a value?

Reply:
Yes. It is initialized in initWithFrame, together with all other values. Also, because it is a 
UIEdgeInsets, it cannot be nil. In the worst case scenario its value would be UIEdgeInsetsZero.

Reply:
I am not worried of the value being nil. My concern is originally we had those constant values set for insets, and since they have been removed, how do we set the initial value for actionsInsets?

Reply:
When I was searching "actionsInsets =" it doesn't seem it was set a default value.

Reply:
The value of self.actionsInsets was not modified in this PR. It already existed in the file, so you many need to expand all the file or open it in RAW mode.
The assignment "self.actionsInsets = UIEdgeInsetsMake(8.0f, 8.0f, 8.0f, 8.0f);" is in the initWithFrame: method.

Reply:
I see. I was surprised to see MDCDialogActionsInsets was not used there. Thanks for pointing it out.